pkg/backoff/backoff.go

```diff
@@ -56,6 +56,11 @@ type Exponential struct {
 	// for logging purposes.
 	Name string
 
+	// BackoffResetDuration is since lastBackoffStart exceeded the duration time that reset the backoff period
+	BackoffResetDuration time.Duration
```

Comment:
Let's name this `BackoffReset` (to be consistent with Min/Max) and document it in terms of how it is used from a user's perspective. For example:

```suggestion
	// BackoffReset will reset the exponential back-off if no attempt is made for the amount of time specified here. 
	// Needs to be larger than the Max duration, otherwise it will be ignored to avoid accidental resets.
	// If unspecified, no reset is performed.
	BackoffReset time.Duration
```